packages/mdc-menu/README.md

```diff
@@ -227,22 +227,22 @@ Method Signature | Description
 `removeClassFromElementAtIndex(index: number, className: string) => void` | Removes the `className` class from the element at the `index` specified.
 `addAttributeToElementAtIndex(index: number, attr: string, value: string) => void` | Adds the `attr` attribute with value `value` to the element at the `index` specified.
 `removeAttributeFromElementAtIndex(index: number, attr: string) => void` | Removes the `attr` attribute from the element at the `index` specified.
-`elementContainsClass(element: HTMLElement, className: string) => boolean` | Returns true if the `element` contains the `className` class.
+`elementContainsClass(element: Element, className: string) => boolean` | Returns true if the `element` contains the `className` class.
 `closeSurface() => void` | Closes the menu surface.
-`getElementIndex(element: HTMLElement) => number` | Returns the `index` value of the `element`.
-`getParentElement(element: HTMLElement) => ?HTMLElement` | Returns the `.parentElement` element of the `element` provided.
-`getSelectedElementIndex(element: HTMLElement) => number` | Returns the `index` value of the element within the selection group provided, `element` that contains the `mdc-menu-item--selected` class.
+`getElementIndex(element: Element) => number` | Returns the `index` value of the `element`.
+`getParentElement(element: Element) => ?Element` | Returns the `.parentElement` element of the `element` provided.
+`getSelectedElementIndex(element: Element) => number` | Returns the `index` value of the element within the selection group provided, `element` that contains the `mdc-menu-item--selected` class.
 `notifySelected(index: number) => void` | Emits a `MDCMenu:selected` event for the element at the `index` specified.
 
 ### `MDCMenuFoundation`
 
 Method Signature | Description
 --- | ---
 `handleKeydown(evt: Event) => void` | Event handler for the `keydown` events within the menu.
-`handleItemAction(listItem: !HTMLElement) => void` | Event handler for list's action event.
+`handleItemAction(listItem: !Element) => void` | Event handler for list's action event.
```

Comment:
Same here - the `!` seems like closure syntax. I think we can remove.

Reply:
Done